index.bs

```diff
@@ -417,6 +451,18 @@ An attribution report is a [=struct=] with the following items:
 
 </dl>
 
+<h3 dfn-type=dfn>Attribution aggregatable histogram</h3>
+
+An attribution aggregatable histogram is a [=struct=] with the following items:
+
+<dl dfn-for="">
+: key
```

Comment:
```suggestion
: <dfn>key</dfn>
```

Reply:
Will add this when adding histogram creation algorithm.